scraper/src/scraper/providers/anthropic.py

```
@@ -12,12 +13,33 @@
 logger = logging.getLogger(__name__)
 
 _URL = "https://www.anthropic.com/pricing"
+_CLAUDE_SONNET_5 = "Claude Sonnet 5"
+
+# Claude Sonnet 5 促進価格の最終適用日（2026-08-31 まで $2/$10、以降 $3/$15）。
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)